examples/fastify/tsconfig.json

```diff
@@ -7,5 +7,6 @@
     "lib": ["esnext", "DOM", "DOM.Iterable"],
     "allowSyntheticDefaultImports": true,
     "skipLibCheck": true
-  }
+  },
+  "include": ["src/*"]
```

Comment:
Had to add this in as tsc was complaining about errors in `graphql-yoga/src/server.ts`
Many errors like this
```
> example-fastify@0.13.6 check /home/arishi/forks/graphql-yoga/examples/fastify
> tsc --pretty --noEmit

../../packages/graphql-yoga/src/server.ts:351:15 - error TS2578: Unused '@ts-expect-error' directive.

351               // @ts-expect-error Add plugins has context but this hook doesn't care
       
```